src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -154,14 +153,22 @@ class StreamGroup extends React.Component<Props, State> {
   }
 
   toggleSelect = (evt: React.MouseEvent<HTMLDivElement>) => {
-    if ((evt.target as HTMLElement)?.tagName === 'A') {
+    const targetElement = evt.target as Partial<HTMLElement>;
```

Comment:
You can probably get away with using `evt.currentTarget` here which simplifies the types a lot.

Reply:
Nah it actually needs to be `target`. Since `currentTarget` is what the event is bound to.

We need target to see if it is something we don't want to bubble up to the `currentTarget`.